src/main/scala/beam/agentsim/agents/ridehail/allocation/PoolingAlonsoMora.scala

```diff
@@ -265,11 +265,16 @@ class PoolingAlonsoMora(val rideHailManager: RideHailManager)
                 scheduleToCache = scheduleToCache :+ orig
                 None
               } else {
+                val personId = orig.person match {
```

Comment:
nitpick: you can use `orig.person.map(_.personId)`

Reply:
replaced code as suggested